bundles/org.openhab.binding.tivo/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="tivo"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="sckt">
+		<label>TiVo DVR</label>
+		<description>Monitor
+			and control your TiVo via DIRECT SOCKET commands leveraging the TIVO protocol 1.1 specification.
+			The TiVo TCP Control Protocol is an ASCII-based command protocol for remote control of a TiVo DVR
+			over a TCP local
```

Comment:
Can you fix the line break?

Reply:
I was fighting with spotless over these. I finally figured out that it works best to make it one long line and then let it fix it. If you try and fix each line that is already broken up, it ends up like this :(